products/chef-infra-server.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Chef Infra Server
+category: server-app
+tags: erlang-runtime ruby-runtime
+iconSlug: chef
+permalink: /chef-infra-server
+alternate_urls:
+-   /chef-server
+versionCommand: chef-server-ctl version
+releasePolicyLink: https://docs.chef.io/versions/
+changelogTemplate: https://docs.chef.io/release_notes_server/#__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   repology: chef-infra-server
+
+auto:
+  methods:
+  -   custom: chef-infra-server
+
+# support(x) = releaseDate(x+1)
+# eol(x) = releaseDate(x+2) or the date documented on https://docs.chef.io/versions/
+releases:
+-   releaseCycle: "15"
+    releaseDate: 2022-06-13
+    support: true
+    eol: false
+    latest: "15.9.20"
+    latestReleaseDate: 2024-01-10
+
+-   releaseCycle: "14"
+    releaseDate: 2020-06-08
+    support: 2022-06-13
+    eol: false
+    latest: "14.16.19"
+    latestReleaseDate: 2022-06-07
+
+-   releaseCycle: "13"
+    releaseDate: 2019-05-31
+    support: 2020-06-08
+    eol: 2021-06-30
```

Comment:
```suggestion
    eol: 2021-06-30 # https://web.archive.org/web/20210919232343/https://docs.chef.io/versions/
```

Reply:
If you're looking for the source fo this date I wrote the blog on it back in 2021: https://www.chef.io/blog/announcing-the-chef-infra-server-13-deprecation

Reply:
Thanks !